src/test-support/lines/text-line.ts

```diff
@@ -56,13 +56,13 @@ export default new TextLine({
           type: 'PerhapsBrokenAway',
         },
         {
-          enclosureType: ['BROKEN_AWAY', 'PERHAPS_BROKEN_AWAY'],
+          enclosureType: [],
           cleanValue: 'ŠA₂',
           value: 'ŠA₂',
           name: 'ŠA',
           nameParts: [
             {
-              enclosureType: ['BROKEN_AWAY', 'PERHAPS_BROKEN_AWAY'],
+              enclosureType: [],
```

Comment:
Why are these removed?

Reply:
i use them in the FragmentAnnotation.test.tsx where they wouldn't be displayed if they are BROKEN so they need not to be broken.

Reply:
Do not break existing tests. The new enclosure type is incorrect given the enclosures in the line.

If you need something else create new data. You can add factories to reduce duplication.

Reply:
ok i added a reading to textline. All tests pass.